script.go

```diff
@@ -934,19 +935,29 @@ func splitHeaderLine(line string) (names []string, pos []int) {
 // The whitespace on the right of the start position (e.g. "1  \t") is trimmed.
 // This of course requires that the table is properly formatted in a way that the
 // header columns are indented to fit the data exactly.
-func splitByPositions(line string, positions []int) []string {
+func splitByPositions(line string, positions []int, splitByTabs bool) []string {
```

Comment:
Question: did you try to add a separate `splitByPositionsWithTabs` function? I wonder if a bit of repetition in place of the `if` might help readability. Same for `joinByPositions.

Reply:
I've updated the commit msg to clarify further and added an example. I considered refactoring but thought maybe it's easier to keep it concise. I'm happy to change it if it is still not very readable after the updated commit msg and example.

Reply:
No need to do that, it was just an idea. We can still rework this later if further changes to the way we split are needed :+1: